packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -114,7 +113,7 @@ export class AddJob {
       })
     );
 
-    const delay = digestAmount ?? delayAmount;
+    const delay = command.filtered ? 0 : digestAmount ?? delayAmount;
```

Comment:
forgot to mention that we have to update some logs here in this file, like to provide the correct information about the delay